Project.toml

```
@@ -55,6 +56,7 @@ NamedArrays = "0.9"
 Reexport = "0.2, 1"
 Requires = "0.5, 1.0"
 SciMLBase = "1.37.1"
+Setfield = "0.7.1, 0.8"
```

Is Setfield 0.7 actually tested and needed? Maybe
```suggestion
Setfield = "0.8"
```
is sufficient?